specification/web/resource-manager/Microsoft.Web/stable/2021-01-01/KubeEnvironments.json

```diff
@@ -0,0 +1,650 @@
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete kube environment by name": {
+            "$ref": "./examples/KubeEnvironments_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "patch": {
+        "tags": [
+          "KubeEnvironments"
+        ],
+        "summary": "Creates or updates a Kubernetes Environment.",
+        "description": "Description for Creates or updates a Kubernetes Environment.",
+        "operationId": "KubeEnvironments_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the Kubernetes Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "kubeEnvironmentEnvelope",
+            "in": "body",
+            "description": "Configuration details of the Kubernetes Environment.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/KubeEnvironmentPatchResource"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/KubeEnvironment"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/KubeEnvironment"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update kube environments": {
+            "$ref": "./examples/KubeEnvironments_Update.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AppLogsConfiguration": {
+      "type": "object",
+      "properties": {
+        "destination": {
+          "type": "string"
+        },
+        "logAnalyticsConfiguration": {
+          "$ref": "#/definitions/LogAnalyticsConfiguration"
+        }
+      }
+    },
+    "ArcConfiguration": {
+      "type": "object",
+      "properties": {
+        "artifactsStorageType": {
+          "enum": [
+            "LocalNode",
+            "NetworkFileSystem"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "StorageType",
+            "modelAsString": false
```

Comment:
Are you sure it is `false`? If `false` you are not allowed to added enum value within same api-version.

Same for other enum.

Reply:
This should be ok for us.